src/components/molecules/Post/Post.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../styles/mixins.scss';
+@import '../../../styles/variables.scss';
+
+.post__field {
+    margin: 1rem 3rem 0 3rem;
```

Comment:
Just a note: if you use flex you could write these styles like 4 times shorter, but that's just a note, i don't want you to refactor that, I don't think that we have time for that